modules/common/src/main/scala/com/snowplowanalytics/snowplow/postgres/streaming/DummyStreamSink.scala

```diff
@@ -12,9 +12,33 @@
  */
 package com.snowplowanalytics.snowplow.postgres.streaming
 
-import cats.effect.{Resource, Concurrent}
+import org.log4s.getLogger
+
+import cats.effect.{Resource, Concurrent, Sync, Timer}
+import cats.effect.concurrent.Ref
+
+import fs2.Stream
+
+import scala.concurrent.duration.FiniteDuration
 
 object DummyStreamSink {
-  def create[F[_]: Concurrent]:Resource[F, StreamSink[F]] =
-    Resource.pure[F, StreamSink[F]](_ => Concurrent[F].pure(()))
+  def create[F[_]: Concurrent: Timer](period: FiniteDuration): Resource[F, StreamSink[F]] =
+    for {
+      counter <- Resource.eval(Ref.of(0))
+      _ <- Resource.make(Concurrent[F].start(reporter(counter, period)))(_.cancel)
+  } yield { _ =>
+    counter.update(_ + 1)
+  }
+
+  lazy val logger = getLogger
+
+  private def reporter[F[_]: Sync: Timer](counter: Ref[F, Int], period: FiniteDuration): F[Unit] =
+    Stream.awakeDelay[F](period)
+      .evalMap(_ => counter.getAndSet(0))
+      .evalMap { count =>
+        if (count > 0) Sync[F].delay(logger.info(s"Discarded $count bad rows"))
```

Comment:
```suggestion
        if (count > 0) Sync[F].delay(logger.info(s"Discarded $count bad rows during last $period"))
```

I think it's just important to tell a user that this amount is generated during this period. For example this situation:

```
Discarded 1000 bad rows
Discarded 1002 bad rows
Discarded 1005 bad rows
Discarded 1010 bad rows
```

Doesn't look as alarming as it should be when user doesn't know that the counter is being resetted.